README.md

```diff
@@ -1 +1,181 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+> A portfolio I built with html and css.
+
+
+## 🛠 Built With <a name="built-with"></a>
+1. html
+2. css
+3. Linters
+
+### Tech Stack <a name="tech-stack"></a>
+1. html
+2. css
+3. Linters
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://styles=css/">CSS</a></li>
+    <li><a href="https://linters/">Linters</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+Added index.html file
+Added style.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Live-demo will br added soon.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Web browser
+Code editor
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+Run this command:
+git clone https://github.com/Kwamay/Hello_Microverse_Project.git
+
+
+
+### Install
+
+Install this project with:
+
+cd my-project
+npm install
+
+
+### Usage
+
+To run the project, execute the following command:
+
+ Open index.html using live server extension.
+
+### Run tests
+
+To run tests, run the following command:
+
+Coming soon
+
+### Deployment
+
+You can deploy this project using:
+
+Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@kwamay](https://github.com/kwamay)
+- Twitter: [@Kwamay3](https://twitter.com/Kwamay3)
+- LinkedIn: [Kwamay Adu](https://linkedin.com/in/кωαмαу-adu-2b396321a)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] Add header
+- [ ] Add body
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project kindly give me stars
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank God for given me the strength and Microverse 
+for providing content for this project. 
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Unfortunately, the link to your MIT license file seems to be broken but the fix is very easy. There is a ```NOTE``` at the bottom of the [Microverse readme template](https://github.com/microverseinc/readme-template) that contains instructions on how to add a license file to your project. I will also paste the note here so you can access easily: 

NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.